# Methods

#### Data Retrieval
Raw sequencing data were obtained from the NCBI Sequence Read Archive (SRA). Accession metadata and associated experimental conditions were compiled into a structured CSV file for downstream use. FASTQ files were retrieved using **SRA Toolkit**

#### Read Quality Control
Initial quality assessment of raw reads was performed with **fastp**, which conducted adapter trimming, quality filtering, and removal of low-quality sequences. Per-sample quality reports generated by fastp were complemented with **FastQC** summaries to evaluate sequence quality, GC content, and duplication profiles. These reports were aggregated using **MultiQC** to provide a unified overview of preprocessing results. Those reports are available at https://github.com/AriannaM-A/Transcriptome-Assembly-TACC/blob/main/Data/multiqc_data.zip

#### Transcriptome Assembly
De novo transcriptome assembly was performed using two independent assemblers to ensure robustness in transcript reconstruction. Trinity and Spades were used to assemble two trascriptomes. These transcriptomes were then benchmarked against the predicted gene models published for *Fragilariopsis cylindrus* (Mock et al., 2017) for a comparison between the transcriptomes and to see which was more biologically complete.

#### Coding Region Identification
Putative open reading frames (ORFs) and coding sequences were identified with **TransDecoder**, which detects likely protein-coding regions using intrinsic sequence metrics and optional homology evidence. Longest ORFs and predicted peptide sequences were extracted for downstream annotation with EnTAP.

#### Transcript Quantification
Transcript abundance estimation was carried out using **Salmon** in quasi-mapping mode. Reads were mapped back to the assembled transcriptome to generate TPM and count matrices for downstream QC and expression analysis.

#### Functional Annotation
Functional annotation was performed with **EnTAP**, which integrates homology searches, protein domain identification, contaminant filtering, and Gene Ontology (GO) assignment. EnTAP was supplied with the predicted TransDecoder peptide dataset.

#### Assembly Completeness Assessment
Completeness of the assembled transcriptomes were evaluated using **BUSCO**. The strameonpiles lineage was chosen for the evaluation and the transcriptomes were compared based on the presence of completed, duplicated, and missing reads.
